data/output/presale.rinkeby.json

```diff
@@ -0,0 +1,6 @@
+{
+  "presaleTemplate": {
+    "address": "0x7e0202db110F04e9bCf81912eefa640a72c8588f",
+    "transactionHash": "0x6fb19adcc079d5f05f94656cb6fed98b07221e19848c153ef138c2057cd56478"
+  }
+}
```

Comment:
I think we can remove the deployed instances until we release v1 of the court fundraising repo